README.md

```diff
@@ -49,7 +49,11 @@ faster.
 When stdin is nil or an empty string, the stdin device will get closed immediately.
 
 The `timeout` parameter speifies the timeout threshold for stderr/stdout reading timeout,
-the stdin writing timeout, and the process waiting timeout, respectively.
+the stdin writing timeout, and the process waiting timeout, respectively. If the timeout
+expires, the sub-process will be killed by first a SIGTERM, and then a SIGKILL.
+
+Note that we do nothing with the child processes of the sub-process (if there are),
```

Comment:
"do nothing" is ambiguous. We should say "do not automatically kill the sub-process".